### ISSUE TITLE
Deprecation Warning - ActiveSupport::TimeWithZone

### ISSUE THREAD

**Issue by pedrocarmona** [ankane/blazer#357]
Hi @ankane ,

The following warning is raise in rails 6 and ruby 2.7 with ruby deprecation warnings turned on:

```
DEPRECATION WARNING: /app/vendor/bundle/ruby/2.7.0/gems/blazer-2.4.2/app/controllers/blazer/base_controller.rb:58: warning: deprecated Object#=~ is called on ActiveSupport::TimeWithZone; it always returns nil
 (called from =~ at /app/vendor/bundle/ruby/2.7.0/gems/blazer-2.4.2/app/controllers/blazer/base_controller.rb:58)
```

The request was to:

queries_controller#show and the params had `start_time` and `end_time` in the query string:
```
https://localhost:3000/blazer/queries/1-my-query?start_time=2021-07-01T00%3A00%3A00Z&end_time=2021-09-30T23%3A59%3A59Z
```

**Comment by ankane** [comment]
Hey @pedrocarmona, thanks for reporting. However, I can't figure out how to reproduce (Ruby 2.7.3, Rails 6.0.4.1, Blazer 2.4.3).

**Comment by pedrocarmona** [comment]
I have this in `config/environments/development.rb`
```
config.active_support.deprecation = :log
```

**Comment by ankane** [comment]
It looks like that's the default for new Rails apps, but I'm still not able to reproduce.

**Comment by ankane** [comment]
Hey @pedrocarmona, should be fixed in 2.6.0+. Think it was from variables ending in `_at` (which are parsed as times). Thanks for reporting!
